docs/src/userguide/language_basics.rst

```diff
@@ -568,6 +570,9 @@ possibilities.
    combinations.  An example is a union of an ``int`` and a ``char*``,
    in which case the pointer value may or may not be a valid pointer.
 
+.. [#2] The conversion will fail if the C array is not stack allocated,
```

Comment:
It seems weird that this should be related specifically to stack allocation. Don't global arrays in the module work as well?